tests/FunctionalTestCase.php

```diff
@@ -4,6 +4,7 @@
 
 use MongoDB\Driver\Command;
 use MongoDB\Driver\Cursor;
+use MongoDB\Driver\Exception\CommandException;
```

Comment:
Small nit, but I usually alphabetize all classes under `MongoDB\Driver` together and then list sub-namespaces afterwards (akin to a directory listing showing all files and then directories at the end). I'm not too worried if this stays as is. We might catch it down the line with code inspection and can always clean things up then.